qa/suites/rgw/notifications/tasks/kafka/test_kafka.yaml

```diff
@@ -1,7 +1,7 @@
 tasks:
 - kafka:
     client.0:
-      kafka_version: 2.6.0
+      kafka_version: 3.8.1
```

Comment:
cc @yuvalif

Reply:
ran basic sanity with 3.8.0 - works ok

Reply:
The mirrors don't seem to go back to 2.6.0, so I pulled it up.